### ISSUE TITLE
[BUG] readable stream part of CsvFormatterStream is in objectMode

### ISSUE THREAD

**Issue by AdriVanHoudt** [C2FO/fast-csv#319]
**Describe the bug**
The whole CsvFormatterStream is set in objectMode while this is only used for the writable part and not the readable part.

Atm this breaks for me as I can't pass CsvFormatterStream to hapi as it doesn't allow me pass a stream in objectMode (https://hapi.dev/api/?v=19.0.5#lifecycle-methods)

**Parsing or Formatting?**

 - [X] Formatting
 - [ ] Parsing 

**To Reproduce**
Atm I'm solving it like this: https://github.com/Salesflare/hapi-csv/blob/cors-and-lifecycle-fix/lib/index.js#L118
Which is forcing the readableStream off objectMode, which seems to work fine.
I also can't pass `formatterOptions.objectMode` as false as it will set the writable stream off of objectMode as well.

**Expected behavior**
It doesn't set the readable stream in object mode (or at least has a config for this)

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. MacOS] macOS
 - OS Version [e.g. Mojave] 10.15.2 Catalina
 - Node Version [e.g. 10.16.0] 12.14.0

**Additional context**

**Comment by kanongil** [comment]
I expect this can be fixed by changing

https://github.com/C2FO/fast-csv/blob/205cb22237a708a9a9cb3ef80422157c0c2a5b18/packages/format/src/CsvFormatterStream.ts#L14

to

```js
super({ writableObjectMode: formatterOptions.objectMode });
```

**Comment by doug-martin** [comment]
Thank you for the including the fix!

The is will be included in `v4.0.3`

**Comment by doug-martin** [comment]
published!
